### PR TITLE
Rename postgres redis params

### PR DESCRIPTION
**Pull request by alvarorm22**
Rename external database to externalPostgresql

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

### CODE REVIEW

**Review by alvarorm22 — COMMENTED** [review]
This config has been tested and works with an external Cloud sql postgres instance

File: customizations/README.md [hunk 1]
```diff
@@ -192,11 +192,11 @@ There are two alternatives when connecting the environment with an external post
     ```
     > Note: One kubernetes secret is going to be created automatically during the installation process with the `externalDatabase.password` and `externalDatabase.adminPassword` that you set in previous lines.
```
Comment (miguelangelmorenochacon) [inline customizations/README.md:193]: You haven't changed the comment ⬆️ 

Maybe a `git grep externalDatabase ` could help 👁️

Comment (alvarorm22) [reply customizations/README.md:193]: Oh, this has been when I merge the latest changes of main, this Readme has been modified in https://github.com/CartoDB/carto-selfhosted-helm/pull/68

**Review by miguelangelmorenochacon — APPROVED** [review]
LGTM

**Review by ManuelLR — APPROVED** [review]
LGTM. But please, wait the :green_circle:

**Pull request merged**